src/runtime/threading_backend.cc

```diff
@@ -37,11 +35,14 @@
 #if defined(__hexagon__)
 #include <dlfcn.h>
 #endif
+#include <algorithm>
+#include <thread>
+#define CURRENT_THREAD_HANDLE (static_cast<std::thread::native_handle_type>(0))
 
 namespace tvm {
 namespace runtime {
 namespace threading {
-
+thread_local int max_concurrency_value = 0;
```

Comment:
Why does this need to be `thread_local`?

Reply:
'max_concurrency_value' is cooperating with the 'ParallelLauncher' which is running the worker thread is a thread_local variable,  so we need to make max_concurrency_value to be a thread_local variable too, then when different thread run 'ParallelLauncher', the different 'ParallelLauncher' can get different value of 'max_concurrency_value'